apps/web/src/components/collectibles/collectible-browser.tsx

```diff
@@ -70,15 +71,46 @@ export default function CollectibleBrowser({
   return (
     <div className={css.root}>
       <Heading className={css.title}>{collectible.title}</Heading>
+      { collectible.previewVideo && 
+        <button onClick={() => setShowVideoCoverImage(!showVideoCoverImage)} className={css.flipButton}>
+          { showVideoCoverImage ? 'show video' : 'show cover'}
+        </button>
+      }
       <div className={css.imageWrapper}>
-        <Image
-          src={collectible.image}
-          loader={cmsImageLoader}
-          width={700}
-          height={700}
-          layout="responsive"
-          objectFit="contain"
-        />
+        { !collectible.previewVideo && 
+          <Image
+            src={collectible.image}
+            loader={cmsImageLoader}
+            width={700}
+            height={700}
+            layout="responsive"
+            objectFit="contain"
+          />
+        }
+        { collectible.previewVideo && 
+          <div className={clsx(css.flipBox, {
+            [css.flip]: showVideoCoverImage
+          })}>
+            <div className={css.flipBoxInner}>
+              <div className={css.flipBoxFront}>
+                <video width="100%" controls>
```

Comment:
Assuming we can ignore the audio in the video, we can [add `muted` attribute](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/media-has-caption.md) to avoid the lint error. Alternatively disable the rule.

```suggestion
                <video width="100%" controls muted>
```